src/lib/components/LighthouseScoresMeta/index.js

```diff
@@ -37,7 +37,7 @@ class LighthouseScoresMeta extends BaseElement {
           this.auditedOn,
         );
       } catch (err) {
-        auditedOnText = auditedOn.toLocaleString();
+        auditedOnText = this.auditedOn.toLocaleString();
```

Comment:
@robdodson @samthor I'm assuming this is the correct change?

Reply:
seems like it.